torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -22,12 +22,11 @@ void RRefContext::destroyInstance(bool ignoreRRefLeak) {
   ctx.checkRRefLeaks(ignoreRRefLeak);
 }
 
-void RRefContext::handleException(const Message& message) {
-  if (message.type() == MessageType::EXCEPTION) {
+void RRefContext::handleException(const utils::FutureError* futErr) {
```

Comment:
Why a const pointer?

If this is done to check if the exception is set in the first place, you could check out `c10::optional`.